### ISSUE TITLE
mininmap is displayed over the leaflet layer menu

### ISSUE THREAD

**Issue by sfeakes** [Norkart/Leaflet-MiniMap#122]
As can be seen in the below example
https://jsfiddle.net/bh201jaj/
When the minimap is at the bottom of the display (ie can't be lowered in the Y index), the default behavior is for it to be shown over the top of the standard Leaflet layer control menu.  I assume this is incorrect and it should be under the leaflet dropdown menu so people can select items.
Any idea on how to change this?

**Comment by robpvn** [comment]
True, it shouldn't behave like that. The easiest workaround is to minimise the minimap with the toggle so that you can access the menu, but of course that doesn't actually solve the problem. I think we'll probably have to write some css for the minimap that makes sure it's behind the layer-control on the Z-axis, whatever depth the layer control is at.

**Comment by madeinspace** [comment]
Has this been fixed? I'm on 3.6.0 and I'm having the exact same problem with tooltip

**Comment by robpvn** [comment]
Nope, it hasn't been fixed yet but it shouldn't be too hard to figure out the CSS stuff if you want to have a crack at it. Sorry about the late response!
